tinker-actions/Makefile

```
@@ -27,11 +27,8 @@ VENV_NAME := venv_$(PROJECT_NAME)
 # Include shared makefile
 include ../common.mk
 
-DOCKER_TAG    := ${DOCKER_REGISTRY}/${DOCKER_REPOSITORY}/${IMG_NAME}
-
-all: 
-	@# Help: Runs build, lint, test stage
-	lint docker-build docker-push
+DOCKER_PATH_PREFIX  := $(shell echo ${DOCKER_TAG} | sed 's/:.*//')
+CREATE_REPO_PREFIX  := $(shell echo ${DOCKER_PATH_PREFIX} | sed 's|^[^/]*/||')
```

I would avoid these sed commands and instead re-build the two variables from values, for example:
```suggestion
DOCKER_PATH_PREFIX  := $(DOCKER_REGISTRY)/$(DOCKER_REPOSITORY)/$(DOCKER_SECTION)/$(DOCKER_IMG_NAME)
CREATE_REPO_PREFIX  := $(DOCKER_REPOSITORY)/$(DOCKER_SECTION)/$(DOCKER_IMG_NAME)
```

---

tinker action and hookOS do not follow the standard makefile flow like other infra-components. In the future, there might be some other changes made to the common.mk which might again impact this.

So i am trying to use just the final output variables of common.mk

---

I don't see how using sed would solve that. You are just assuming that the structure of the those variable are kept the same.
IMO, re-constructing the variables like I'm suggesting makes the makefile much clearer, and cleaner. Anyway, my comment is not strong, and I'm fine to keep it as you prefer.